collectors/graph_collector.go

```diff
@@ -366,6 +402,27 @@ func (g *GraphCollector) collectRoutingPolicyMetrics(
 
 		feeBaseStats = newStatsCompiler(numEdges)
 		feeRateStats = newStatsCompiler(numEdges)
+
+		inboundFeeBaseStats = prometheus.NewHistogramVec(
+			prometheus.HistogramOpts{
+				Name: inboundFeeBaseName,
+				Help: inboundFeeBaseHelp,
+				Buckets: prometheus.ExponentialBuckets(
+					1, 2, 20, // 1 to 1_048_576 msat
+				),
+			},
+			[]string{inboundFeeSignLabel},
+		)
+		inboundFeeRateStats = prometheus.NewHistogramVec(
+			prometheus.HistogramOpts{
+				Name: inboundFeeRateName,
+				Help: inboundFeeRateHelp,
+				Buckets: prometheus.ExponentialBuckets(
+					1, 2, 13, // 1 to 8192 PPM
```

Comment:
Should we increase this upper range? IIUC this goes up to 0.80% or 80 bps?

Reply:
:+1: I put some random number here initially, but just checked that the 99-percentile at the moment is 10000 PPM. I bumped it up to an exponent of 15, giving 32768 PPM (~3 %).